import-service/serverless.yml

```diff
@@ -16,9 +23,43 @@ provider:
         - "s3:*"
       Resource:
         - "arn:aws:s3:::products-bucket-aws-epam/*"
-        
-plugins:
-  - serverless-offline
+    - Effect: "Allow"
+      Action: 
+        - "sqs:*"
+      Resource:
+        - Fn::GetAtt: [catalogItemsQueues, Arn]
+    - Effect: "Allow"
+      Action: "sns:*"
+      Resource:
+        Ref: createProductSnsTopic
+
+resources:
+  Resources:
+    catalogItemsQueues:
+      Type: "AWS::SQS::Queue"
+      Properties:
+        QueueName: catalog-items-queue
+    createProductSnsTopic:
+      Type: "AWS::SNS::Topic"
+      Properties:
+        TopicName: create-product-topic  
+    SNSSubscription:
+      Type: "AWS::SNS::Subscription"
+      Properties:
+        Endpoint: maksym.musiichuk@gmail.com
+        Protocol: email
+        TopicArn:
+          Ref: createProductSnsTopic
+    SNSSubscriptionWithFilterPolicy:
+      Type: AWS::SNS::Subscription
+      Properties:
+        Endpoint: aws.musiichuk.maksym@gmail.com
+        Protocol: email
+        FilterPolicy:
```

Comment:
Does filtration really works without specifying MessageAttributes?
Could you please attach screenshots of your emais.